src/component/RecipeFormModal.js

```diff
@@ -45,7 +44,7 @@ const HANDLE_INGREDIENT_INPUT = "handle_ingredient_input";
 function RecipeFormModal({ onClose }) {
   const [isLoading, setIsLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState(null);
-  const [isSuccessful, setIsSuccessful] = useState(null);
+  const [successMessage, setSuccessMessage] = useState(null);
   const [state, dispatch] = useReducer(recipeFormReducer, initialFormValue);
```

Comment:
I think you can name something else instead of `state`, maybe `formState` for better readability?